h2o-genmodel/src/main/java/hex/genmodel/algos/ensemble/StackedEnsembleMojoModel.java

```diff
@@ -0,0 +1,44 @@
+package hex.genmodel.algos.ensemble;
+
+import hex.genmodel.MojoModel;
+
+public class StackedEnsembleMojoModel extends MojoModel {
+
+    MojoModel _metaLearner; //Currently only a GLM. May change to be DRF, GBM, XGBoost, or DL in the future
+    MojoModel[] _baseModels; //An array of base models
+    int _baseModelNum; //Number of base models
+
+    public StackedEnsembleMojoModel(String[] columns, String[][] domains, String responseColumn) {
+        super(columns, domains, responseColumn);
+    }
+
+    @Override
+    public double[] score0(double[] row, double[] preds) {
+        double[] basePreds = new double[_baseModelNum]; //Proper allocation for binomial and regression ensemble (one prediction per base model)
```

Comment:
Allocating memory per-row is not ideal however I understand there are no good solutions here

Reply:
Yes, I could not think of a better way given the current API. But worth modifying in the short future, if possible.